egs2/mini_an4/asr1/run.sh

```diff
@@ -9,5 +9,5 @@ set -o pipefail
     --lang en \
     --train_set train_nodev \
     --valid_set train_dev \
-    --test_sets "train_dev test test_seg" \
+    --test_sets "test test_seg" \
```

Comment:
I guess we don't need to use all sets for integration tests? It may reduce decoding for some tests (with LM notably).

We can directly modify the recipe if it's only used in integration tests or overwrite the option in integration tests scripts.